### ISSUE TITLE
Bug - Accessibility: Cannot tab out of carousel section of homepage

### ISSUE THREAD

**Issue by data-sync-user** [MozillaFoundation/foundation.mozilla.org#14806]
If you try to use keyboard only on NP homepage, you get stuck in the carousel and can’t get to the next section. Can we explore how to make this more keyboard accessible?

┆Issue is synchronized with this [Jira Bug](https://mozilla-hub.atlassian.net/browse/TP1-3284)

**Comment by data-sync-user** [comment]
➤ Kevin Duong commented:

up for review
